examples/session/Rocket.toml

```diff
@@ -1,3 +1,6 @@
+[dev]
+secret_key = "itlYmFR2vYKrOmFhupMIn/hyB6lYCCTXz4yaQX89XVg="
```

Comment:
Let's set this globally instead:

```toml
[global]
secret_key = "itlYmFR2vYKrOmFhupMIn/hyB6lYCCTXz4yaQX89XVg="
```

Reply:
I've updated the configuration. Made secret_key global, removed unneeded staging settings.